### PR TITLE
fix problem with overriding paths

### PR DESCRIPTION
**Pull request by ifiokjr**
This PR allows the yarn workspace paths to co-exist with custom paths in the tsconfig.json file. 

See #3 for more information.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -55,8 +56,18 @@ function init(_modules: { typescript: typeof ts_module }) {
           info.project.getCompilerOptions = function() {
             let oldOptions = oldCOptions.call(this);
             if (!patchedOptions) {
+              let oldPaths: ts_module.MapLike<string[]> =
+              Object.keys(oldOptions.paths || {}).reduce((prev, curr) => {
+                let keys = oldOptions.paths[curr].map((p: string) =>
+                  path.resolve(initialRootPath, oldOptions.baseUrl, p)
+                );
+                return Object.assign({}, prev, { [curr]: keys });
+              }, {}) || {};
```
Comment (ifiokjr) [inline src/index.ts:65]: Doesn't need the `||` statement.
```suggestion
              }, {});
```

**Comment by ifiokjr** [comment]
Closing as this actually doesn't solve the problem.

**Comment by spion** [comment]
Sorry I forgot about this PR! Can you tell me why it didn't solve the problem? It actually looks good to me on a first glance...

**Comment by jquense** [comment]
i also would like to know we have this problem :(

**Comment by jquense** [comment]
(happy to help)

**Pull request closed without merging**